RoleCard/RoleCard.css

```diff
@@ -0,0 +1,39 @@
+.application-container{
+    display: flex; 
+    flex-wrap: wrap; 
+    gap: 1rem; 
+    justify-content: center;
+    font-family: 'Poppins', sans-serif;
+}
+
+.roleCard{
+    font-family: 'Poppins', sans-serif;
+    width: 460px;
+    margin-bottom: 20px;
+    font-weight: 500;
+    flex-shrink: 0;
+}
+
+.roleTitle {
```

Comment:
The figma design shows a slightly larger margin/padding between the role title and the "apply" button.